manta-js/examples/asset-webpack-ts/index.ts

```diff
@@ -5,10 +5,12 @@ import { web3Accounts, web3Enable, web3FromSource } from '@polkadot/extension-da
 
 async function main() {
     await toPrivateOnlySignTest();
-    await toPrivateTest();
-    await privateTransferTest();
-    await toPublicTest();
-    await publicTransferTest();
+    // await toPrivateTest();
```

Comment:
remove comments

Reply:
> Changes from `Development` to `Production` endpoint.
> 
> I usually use `Development` for testing, but its not hard to change them if there is benefit to making `Production` the default

My idea was to have it easier to run the example (they don't need to setup a local network)
But then again if you're running the example you're probably doing some development on a dev environment, so I'll just make it dev.

Reply:
Actually I'll also make it Calamari, because the inconsistency between the assets on Dolphin productiona nd Dolphin develponment is confusing